forms-flow-web/src/components/Draft/table.js

```diff
@@ -57,8 +59,14 @@ function timeFormatter(cell) {
   return <label title={cell}>{localdate}</label>;
 }
 
-const nameFormatter = (cell) => {
-  const name = startCase(cell);
+const nameFormatter = (cell, row) => {
+  let employee = '';
+  if (cell.toLowerCase().includes('sl review')) {
```

Comment:
Sooner or later, we need to define env and constant for the form names and rely on that instead of hardcoding the names.

Reply:
That make sense, That's why I did use .includes() for now. Later we will update it.